lib/utils/routers.dart

```diff
@@ -40,6 +41,21 @@ final router = GoRouter(
       ),
       routes: [],
 
+      /// TODO: Add a sub routes in the future
+    ),
+    GoRoute(
```

Comment:
why create a new absolute route/path here, isn't this supposed to be a sub_route of `PokemonFavoritesConnector`? 🤔